CRM/Core/DAO/Note.php

```diff
@@ -220,11 +234,43 @@ public static function &fields() {
           ],
           'add' => '1.1',
         ],
+        'note_date' => [
+          'name' => 'note_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Note Date'),
+          'description' => ts('Date attached to the note.'),
+          'where' => 'civicrm_note.note_date',
+          'default' => 'CURRENT_TIMESTAMP',
+          'table_name' => 'civicrm_note',
+          'entity' => 'Note',
+          'bao' => 'CRM_Core_BAO_Note',
+          'localizable' => 0,
+          'html' => [
+            'type' => 'Select Date',
+            'formatType' => 'activityDateTime',
+          ],
+          'add' => '5.36',
+        ],
+        'created_date' => [
+          'name' => 'created_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Created Date'),
+          'description' => ts('When the note was created.'),
+          'required' => TRUE,
+          'where' => 'civicrm_note.created_date',
+          'default' => 'CURRENT_TIMESTAMP',
+          'table_name' => 'civicrm_note',
+          'entity' => 'Note',
+          'bao' => 'CRM_Core_BAO_Note',
+          'localizable' => 0,
+          'add' => '5.36',
+        ],
         'modified_date' => [
           'name' => 'modified_date',
           'type' => CRM_Utils_Type::T_TIMESTAMP,
-          'title' => ts('Note Modified By'),
+          'title' => ts('Modified Date'),
           'description' => ts('When was this note last modified/edited'),
+          'required' => TRUE,
```

Comment:
I'm not sure about adding required here - that would impact what needs to be submitted at the apiv4 level I think (@colemanw to confirm)

I think it's OK just relying on the default